api/routers/edr.py

```diff
@@ -286,7 +286,8 @@ async def get_data_area(
         )
 
     request = dstore.GetObsRequest(
-        spatial_area=dstore.Polygon(
+        filter=dict(parameter_name=dstore.Strings(values=parameter_name if parameter_name else None)),
```

Comment:
I think something went wrong with the rebase here. `parameter_name` is now added twice, see line 298.

Reply:
Right! I was a bit unsure of that line. Will remove it ...